src/languageSetup.ts

```diff
@@ -67,7 +67,13 @@ export async function activateLanguageServer(context: vscode.ExtensionContext, s
     status.dispose();
     
     const startScriptPath = customPath || path.resolve(langServerInstallDir, "server", "bin", correctScriptName("kotlin-language-server"));
-    const options = { outputChannel, startScriptPath, tcpPort, env };
+
+    const storagePath = context.storagePath
```

Comment:
Just to clarify, where does this generally point? Is it some VSCode-specific extension-specific application data directory?

Reply:
From the jsdoc:

> An absolute file path of a workspace specific directory in which the extension can store private state. The directory might not exist on disk and creation is up to the extension. However, the parent directory is guaranteed to be existent.

On Windows this translates to something like `C:\Users\<your-user>\AppData\Roaming\Code\User\workspaceStorage\<workspace-id>\fwcd.kotlin`.